newIDE/app/src/ExportAndShare/GenericExporters/HTML5Export.js

```diff
@@ -49,80 +52,85 @@ export const ExplanationHeader = () => {
   );
 };
 
+type HTML5ExportFlowProps = {|
+  ...ExportFlowProps,
+  exportPipelineName: string,
+|};
+
+export const ExportFlow = ({
+  disabled,
+  launchExport,
+  isExporting,
+  exportPipelineName,
+  exportStep,
+}: HTML5ExportFlowProps) =>
+  exportStep !== 'done' ? (
+    <Line justifyContent="center">
+      <RaisedButton
+        label={
+          !isExporting ? (
+            <Trans>Export as a HTML5 game</Trans>
+          ) : (
+            <Trans>Exporting...</Trans>
+          )
+        }
+        primary
+        id={`launch-export-${exportPipelineName}-button`}
+        onClick={launchExport}
+        disabled={disabled || isExporting}
+      />
+    </Line>
+  ) : null;
+
 export const DoneFooter = ({
   renderGameButton,
 }: {|
   renderGameButton: () => React.Node,
-|}) => (
-  <Column noMargin>
-    <Text>
-      <Trans>
-        You can now upload the game to a web hosting to play to the game.
-      </Trans>
-    </Text>
-    <AlertMessage kind="warning">
-      <Trans>
-        Your game won't work if you open index.html on your computer. You must
-        upload it to a web hosting (Kongregate, Itch.io, etc...) or a web server
-        to run it.
-      </Trans>
-    </AlertMessage>
-    <Spacer />
-    {renderGameButton()}
-    <Spacer />
-    <FlatButton
```

Comment:
cleaned up all those links, they are all displayed on the wiki page now linked under 1 button.